test/testmachinery/shoots/operations/operations.go

```diff
@@ -87,7 +91,13 @@ var _ = ginkgo.Describe("Shoot operation testing", func() {
 		guestBookTest.WaitUntilGuestbookDeploymentIsReady(ctx)
 		guestBookTest.Test(ctx)
 
-	}, hibernationTestTimeout)
+	}, hibernationTestTimeout, framework.WithCAfterTest(func(ctx context.Context) {
+		if isShootHibernated && v1beta1helper.HibernationIsEnabled(f.Shoot) {
+			ginkgo.By("Wake up shoot")
+			err := f.WakeUpShoot(ctx)
+			framework.ExpectNoError(err)
+		}
+	}, 25*time.Minute))
```

Comment:
A minor suggestion to extract `25*time.Minute` into a constant, much like `hibernationTestTimeout`. I believe it will bring further clarification and save time looking up the docs what's this `duration` about. WDYT ?

Reply:
Just looked at the updates, thanks for extracting the duration into a constant.

/lgtm